fenix/app/build.gradle

```diff
@@ -206,7 +206,11 @@ android {
 
             reset()
 
-            include "x86", "armeabi-v7a", "arm64-v8a", "x86_64"
+            if (project.hasProperty("benchmarkTest")) {
+                include "arm64-v8a"
```

Comment:
A nit about this style of change is that we are making the app module configuration dependent on the benchmark module. The benchmark module is supposed to be testing against a release module and changing our configuration can lead us to make changes where we are no longer benchmarking against the correct app module configuration.

If we can avoid changing the app module, when adding the benchmark one, that would be the ideal environment where the two are not directly related at all.

Reply:
This is needed for generator and benchmark tests both - when this property is present in the gradle command, then only `arm64-v8a apk` is created since we don't need others. However this isn't done as an optimisation 😄 but gradle wasn't able to pick the right apk and would fail trying to install x86 or other APKs when running benchmark or the baseline profile generator. At this moment, we don't have solution to this. 

Even though the GMD is arm64, gradle doesn't get that and tries to just install the first APK it finds in the builds folder - which can be x86 variant. Seems like an issue for projects with apk splits but wouldn't happen on other projects. 

Since we run locally and on ARM64 devices, this was added. For CI, we might have to update this to x86 instead but would need this condition, unless there is a different approach that I haven't been able to figure out.

Reply:
> gradle wasn't able to pick the right apk and would fail trying to install x86 or other APKs when running benchmark or the baseline profile generator. At this moment, we don't have solution to this.

This sounds like a useful comment on why we are doing this that we should add here, i.e. there is a bug somewhere in gradle or GMD, and we are writing this workaround that should be removed when its fixed. Is that right?

Reply:
Yeah, good point! I'll add this as a comment! 👍

Reply:
Updated! and updated the generate and startup benchmark docs to mention the steps!